docs/sources/setup-grafana/upgrade-grafana.md

```diff
@@ -13,33 +13,33 @@ weight: 500
 
 # Upgrade Grafana
 
-We recommend that you upgrade Grafana often to stay up to date with the latest fixes and enhancements.
-In order to make this a reality, Grafana upgrades are backward compatible and the upgrade process is simple and quick.
+We recommend that you upgrade Grafana often to stay current with the latest fixes and enhancements.
+Because Grafana upgrades are backward compatible, the upgrade process is straightforward. Upgrading between many minor versions and one major version is generally safe, and dashboards and graphs will not change.
 
-Upgrading between many minor versions and one major version is generally safe, and dashboards and graphs will look the same.
-There might be minor breaking changes in some releases.
-We outline these in the [What's New overviews]({{< relref "../whatsnew/" >}}) for each release.
-For versions of Grafana prior to v9.2, we also published additional information in the [Release Notes]({{< relref "../release-notes/" >}}).
-We also list all changes, with links to pull requests or issues when available, in the [Changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md).
+In addition to common tasks you should complete for all versions of Grafana, there might be additional upgrade tasks to complete for a version.
 
-## Backup
+> **Note:** There might be minor breaking changes in some releases. We outline these in the [What's New ]({{< relref "../whatsnew/" >}}) document for each release.
 
-We recommend that you backup a few things in case you have to rollback the upgrade.
+For versions of Grafana prior to v9.2, we published additional information in the [Release Notes]({{< relref "../release-notes/" >}}).
 
-- Installed plugins - Back them up before you upgrade them in case you want to rollback the Grafana version and want to get the exact same versions you were running before the upgrade.
-- Configuration files do not need to be backed up. However, you might want to in case you add new configuration options after upgrade and then rollback.
+When available, we list all changes with links to pull requests or issues in the [Changelog](https://github.com/grafana/grafana/blob/main/CHANGELOG.md).
 
-### Database backup
+> **Note:** When possible, we recommend that you test the Grafana upgrade process in a test of development environment.
```

Comment:
"in a test **or** development environment"

Reply:
Good catch, thank you!